glossary.md

```diff
@@ -93,6 +94,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | evaluation metric   | phép đánh giá                                                  |                       |
 | example             | mẫu                                                            |                       |
 | explicit feedback   | phản hồi trực tiếp                                             |                       |
+| expression (math)   | diễn tả toán học                                               |                       |
```

Comment:
```suggestion
| expression (math)   | biểu thức                                              |                       |
```
Ủa hình như trong toán mình có từ `biểu thức` rồi đúng không ta?
Nếu đúng là vậy thì mình cũng không chắc có nên thêm từ này vô glossary không nữa.